src/ui/view/home.rs

```diff
@@ -141,35 +146,31 @@ fn unsecured_fund_view<'a>(
 }
 
 /// render balance card from a tuple: (active, inactive)
-fn balance_view<'a, T: 'a>(balance: &(u64, u64)) -> Container<'a, T> {
+fn balance_view<'a, T: 'a>(ctx: &Context, balance: &(u64, u64)) -> Container<'a, T> {
+    let active_balance = ctx.converter.converts(balance.0);
+    let inactive_balance = ctx.converter.converts(balance.1);
     let col = Column::new()
         .push(text::bold(text::simple("Balance:")))
         .push(
             Row::new()
                 .padding(5)
                 .push(Container::new(text::simple("active")).width(Length::Fill))
                 .push(
-                    Container::new(text::bold(text::simple(&format!(
-                        "{}",
-                        balance.0 as f64 / 100000000_f64
-                    ))))
-                    .width(Length::Shrink),
+                    Container::new(text::bold(text::simple(&format!("{}", active_balance))))
+                        .width(Length::Shrink),
                 )
-                .push(text::simple(" BTC")),
+                .push(text::simple(&format!(" {}", ctx.converter.unit))),
```

Comment:
nit: `format!` is not necessary, you can use `to_string`:
```
.push(text::simple(&ctx.converter.unit.to_string())),
```

Reply:
(If you choose to use `to_string` instead, please update it in the other parts of the code where there is `format!("{}", something)`, so we remain consistent; feel free to discard this suggestion though :))

Reply:
Thank you I will create an issue